### DISCUSSION TITLE
Bug Discussions!🪲

### DISCUSSION THREAD

**Discussion by korenbenezra** [Koren-Ben-Ezra/Exams#4] [Bug Discussions🪲]
##  🪲Welcome to the Bug Discussions!🪲
_**Noticed a bug?! Write it here!**_

**Comment by CodeSlayer10** [comment]
just to clarify, isn't your solution to 2017 86 actually O(n^2) rather than O(n)?

**Reply by korenbenezra** [reply]
Yes, fixed
